bin-ranges-transfer/src/main/java/uk/gov/pay/java_lambdas/bin_ranges_transfer/DependencyFactory.java

```diff
@@ -0,0 +1,69 @@
+
+package uk.gov.pay.java_lambdas.bin_ranges_transfer;
+
+import org.apache.sshd.client.ClientBuilder;
+import org.apache.sshd.client.SshClient;
+import org.apache.sshd.client.keyverifier.AcceptAllServerKeyVerifier;
+import org.apache.sshd.client.session.ClientSession;
+import org.apache.sshd.common.config.keys.FilePasswordProvider;
+import org.apache.sshd.common.config.keys.loader.KeyPairResourceLoader;
+import org.apache.sshd.common.keyprovider.KeyIdentityProvider;
+import org.apache.sshd.common.util.security.SecurityUtils;
+import org.apache.sshd.sftp.client.SftpClient;
+import org.apache.sshd.sftp.client.SftpClientFactory;
+import software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider;
+import software.amazon.awssdk.core.SdkSystemSetting;
+import software.amazon.awssdk.http.crt.AwsCrtHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.ssm.SsmClient;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.security.GeneralSecurityException;
+import java.security.KeyPair;
+import java.util.Collection;
+
+/**
+ * The module containing all dependencies required by the {@link App}.
+ */
+public class DependencyFactory {
+
+    private DependencyFactory() {
+    }
+    
+    public static SsmClient ssmClient() {
+        return SsmClient.builder()
+            .credentialsProvider(EnvironmentVariableCredentialsProvider.create())
+            .region(Region.EU_WEST_1)
```

Comment:
I'd suggest putting this into `Constants` - it seems equivalent to other items in there (things that are repeated and are unlikely to change).

Reply:
done, have made the bucket an env value also